tests/testthat/test-Integrate.R

```diff
@@ -0,0 +1,35 @@
+#--- Copula tests ------------------------------------------------------
+
+## library(LocalCop)
+## library(TMB)
+## library(testthat)
+
+
+# TMB version of the function
+integrated_exp_tmb <- TMB::MakeADFun(data = list(model = "exponential_integral", lambda = 1, lower = 0, upper = 1), parameters = list(), silent = TRUE)
```

Comment:
now you would have `data = list(model = "exponential_integral")` and `parameters = list(lambda = 1, lower = 0, upper = 1)`.